index.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+const RunKitTask = require('runkit-task');
+const async = require('async');
+const fs = require('fs-extra');
+
+class CleanTask extends RunKitTask {
+  get description() {
+    return 'Cleans out the indicated directory';
+  }
+
+  // save the filename:hashname map to file:
+  execute(allDone) {
```

Comment:
couldn't we use process and then it's called for each item? then it would be 

`items: [ 'dir1', 'dir2' ]`